src/main/java/org/takes/http/MainRemote.java

```diff
@@ -49,6 +49,12 @@
 public final class MainRemote {
 
     /**
+     * The format of the error message indicating that the file could not be
+     * deleted.
+     * @checkstyle LineLength (2 lines)
+     */
+    private static final String DEL_FILE_ERR_MSG = "The file '%s' could not be deleted";
```

Comment:
@yegor256 because it is the exact same error, exact same context, exact same method even exact same file so for me it has nothing to do with your blog post that I know already BTW, but still OK will try to give different messages anyway.